src/components/Avatar/tests/Avatar.test.tsx

```diff
@@ -116,10 +115,6 @@ describe('<Avatar />', () => {
   });
 
   describe('styleClass', () => {
-    it('defaults to five styles', () => {
```

Comment:
This is testing an internal implementation detail, I didn't want to expose STYLE_CLASSES so it was easiest to remove this test instead of moving files around to avoid exposing STYLE_CLASSES